worldedit-core/src/main/java/com/sk89q/worldedit/function/block/SnowSimulator.java

```diff
@@ -66,6 +69,13 @@ public boolean isGround(BlockVector3 position) {
             return true;
         }
 
+        // Slabs are also valid but not "FullCube"
+        BlockCategory blockCategorySlabs = new BlockCategory(BlockCategories.SLABS.getId());
```

Comment:
shouldn't be constructing a new instance. `BlockCategories` exists.

Reply:
Yep, sorry for that, my bad.